cmd/cli/generateKey.go

```diff
@@ -32,7 +32,7 @@ func GenerateFlow(password, output, identifier string, bits int) {
 		panic(fmt.Sprintf("Error creating key: %s\n", err))
 	}
 
-	fingerPrint, _ := remote_signer.GetFingerPrintFromKey(key)
+	fingerPrint, _ := tools.GetFingerPrintFromKey(key)
```

Comment:
The same problem of previous comment about error handling.

Reply:
Same here, the key is guaranteed to be valid. So fingerprint will always return.

Reply:
Okay!